project-base/app/AppKernel.php

```diff
@@ -48,6 +50,8 @@ public function registerBundles()
             new Shopsys\FrameworkBundle\ShopsysFrameworkBundle(),
             new Shopsys\ReadModelBundle\ShopsysReadModelBundle(), // has to be loaded after ShopsysFrameworkBundle because it overrides Twig `image` function
             new Shopsys\ShopBundle\ShopsysShopBundle(), // must be loaded as last, because translations must overwrite other bundles
```

Comment:
Update the comment here as it is not loaded as last.

Reply:
fixed by alphabetical order